os/drivers/ai-soc/ndp120/src/ndp120_api.c

```diff
@@ -629,10 +638,14 @@ static int configure_audio(struct syntiant_ndp_device_s *ndp, unsigned int pdm_i
 		goto errout_configure_audio;
 	}
 
-	audio_frame_bytes_ = misc_config.audio_frame_step
-		* SYNTIANT_NDP120_AUDIO_SAMPLES_PER_WORD;
+	*sample_size_orig_annot = misc_config.audio_sample_size_bytes;
```

Comment:
```
configure_audio(dev, DMIC_768KHZ_PDM_IN_SHIFT_FF, AUDIO_TANK_MS, &dev->sample_size, &dev->sample_size_orig_annot); //use dev instead of dev->ndp
...
dev->sample_size_orig_annot = misc_config.audio_sample_size_bytes;
```